### PR TITLE
Flip all jobs for mozilla-aurora to Marionette (#576)

### PR DESCRIPTION
**Pull request by whimboo**
I still have to test if everything works. But @jmaher if you find some time and can have a look it would be great. Not sure if @bobsilverberg will be around today.

### CODE REVIEW

**Comment by bobsilverberg** [comment]
I have looked over the diffs and nothing jumps out at me as a problem, although there is plenty of stuff in there that I do not understand. r+, assuming it all is fully tested

File: config/staging/jenkins.patch [hunk 1]
```diff
@@ -256,51 +256,12 @@ index 2cec9ac..a8c78ed 100644
...
-diff --git a/jenkins-master/jobs/mozilla-aurora_l10n/config.xml b/jenkins-master/jobs/mozilla-aurora_l10n/config.xml
-index fd615c3..387e40c 100644
---- a/jenkins-master/jobs/mozilla-aurora_l10n/config.xml
-+++ b/jenkins-master/jobs/mozilla-aurora_l10n/config.xml
-@@ -4,7 +4,7 @@
-   <description>Execute l10n tests for Aurora builds.</description>
+   <description>Execute functional tests for Firefox Developer Edition builds.</description>
    <logRotator class="hudson.tasks.LogRotator">
      <daysToKeep>-1</daysToKeep>
 -    <numToKeep>10</numToKeep>
```
Comment (jmaher) [inline config/staging/jenkins.patch]: why are we not keeping any of these aurora l10n logs?

**Comment by jmaher** [comment]
overall, I just had the one question above.  I, like bobsilverberg, am not understanding all the changes.  It looks like a lot of stuff!  Nothing jumps out at me to block deploying this.

File: config/staging/jenkins.patch (hunk 1 above)
Comment (whimboo) [inline config/staging/jenkins.patch]: We do no longer run l10n tests for aurora. We can restart with those once we have a better strategy on them. As result I removed the job from the config.

**Pull request merged**